doc/manual/source/interaction/async-tasks.rst

```diff
@@ -1,37 +1,37 @@
-Web-API Asynchronous tasks 
+Web-API Asynchronous tasks
 ==========================
 
-Some Web-API commands can run for a long time depending on how much data they're dealing with, and hence
-are executed asynchronously. Meaning that once invoked, they start the job in a background thread and return immediately.
+Some Web-API commands can run for a long time depending on how much data they're dealing with, and hence,
+are executed asynchronously, meaning that, once invoked, they start the job in a background thread and return immediately.
 
 The response contains a unique identifier for the asynchronous task and information on the task's running status.
-This task identifier can then be used to poll the status of the task by querying the ``/tasks/<task id>`` API end point.
+This task identifier can then be used to poll the status of the task by querying the ``/tasks/<task id>`` API endpoint.
 
-If no task is given (e.g. by querying the ``/tasks`` endpoint), a list of all available tasks will be returned.
+If no task is given, e.g. by querying the ``/tasks`` endpoint, a list of all available tasks will be returned.
 
-A running task can be canceled with the ``cancel=true`` argument. For example: 
+A running task can be canceled with the ``cancel=true`` argument. For example:
 
 ::
 
    http://localhost:8182/tasks/3?cancel=true
 
-This request will ask the operation to cancel and return immediately. Note however that the task may take a while until the running task
-is actually canceled, as it may be blocking on I/O or being temporarily in a non cancelable state. Polling the task repeteadly with the
-``cancel=true`` parameter is not a problem though.
+This request will ask the operation to cancel and return immediately. Note, however, that the task may remain a while until the running task
```

Comment:
This sounds a little awkward. Perhaps use:

`Note, however, that it may take some time for the cancellation request to complete, as the running task may be blocking on I/O or in a temporarily non cancelable state.`